operations/research-and-development/product/growth/ab-testing.md

```diff
@@ -0,0 +1,119 @@
+
+#### Example 
+The Product Team wants to increase the proportion of users that invite members within the first 14 days on the platform. They’re evaluating two potential UI changes (interventions) to affect this change: 
+1. Test 1: Placing an icon at the top of the left-hand sidebar channel switcher.
+2. Test 2: Placing a banner button at the bottom of the left-hand sidebar channel switcher. 
+3. Control: Hold the current UI constant.
+
+The Product Team is targeting at least a 20% relative increase in the proportion of users that invite members within the first 14 days on the platform. This desired relative change in the proportion of users to “convert” will help govern our calculations for the minimum required sample size, and transitively the amount of time the A/B test will have to run in order to collect the minimum required sample size for each group (test 1, test 2, and the control group).
+
+### Sample Size
+The next step in formulating an A/B test involves determining the minimum required sample size to produce statistically significant results i.e. adequate confidence level and statistical power.
+
+This step is key. It does not need to be done in advance, although it will help you better understand how long the test will need to be run in order to achieve the desired statistical power (Default = .8) and confidence level (Default = .95). 
+
+The statistical power (β), the confidence level (1-𝜶), the minimum detectable effect size (𝝈), the desired test outcome, as well as the historical measure benchmark (as a buffer for the control outcome) are all required to calculate the minimum required sample size. As the test progresses, the calculation for minimum required sample size will change as you’re able to substitute hypothesized outcomes with actual results. The greater the change between groups, the smaller the sample size needed.
+
+These variables and assumptions are listed below, as well as the formula for calculating the sample size for varying types of tests (1-tailed vs. 2-tailed).
+Variables & Assumptions
+n = Sample Size
+This represents the sample size of each group. Ideally, the samples will be the same size for each test group and the control.
+The sample size must be sufficiently large in order for the A/B test to have adequate statistical power (the typical statistical power for A/B tests is .8, which is 1 - β)
+
+𝜶 = Alpha: Uncertainty Level (Probability of Committing Type I Error)
+* Default = .05
+  * Meaning 1/20 chance there is an effect, but you conclude there is no effect
+
+𝜷 = Beta: (Probability of Committing Type II Error)
+* 1 - β = Statistical Power
+
+𝞺1 = Proportion of group 1 to convert
+
+𝞺2 = Proportion of group 2 to convert
+
+𝝈 = |𝞺1 - 𝞺2|
+* Absolute difference between proportion of group 1 and group 2
+
+Z = Z Score
+* Z Score dependent on the specified 𝜶 (alpha) & 𝜷 (beta) statistics
+
+One-sided Test Sample Size Formula: 
+* n = ( (Z1-𝜶 * √ 𝞺1*(1- 𝞺1) ) + (Z1-𝜷 * √ 𝞺2*(1- 𝞺2) ) ) / 𝝈 )2 
```

Comment:
is there a link to an online calculator where you can input the values to get the sample size? (nice-to-have but would reduce user error)

Reply:
Added in the last commit. @jasonblais